storage/src/main/java/tech/pegasys/teku/storage/server/ChainStorage.java

```diff
@@ -158,11 +166,26 @@ public SafeFuture<Optional<SignedBeaconBlock>> getFinalizedBlockAtSlot(final UIn
     return SafeFuture.of(() -> database.getFinalizedBlockAtSlot(slot));
   }
 
+  @Override
+  public SafeFuture<Void> onNoBlobsSlot(final SlotAndBlockRoot slotAndBlockRoot) {
```

Comment:
I don't want something like NoBlobSidecars because we could have several BlobSidecars in block, so such name is less clear for me. 
Renaming back manager, ok, I will make separate rename PR for it.